### ISSUE TITLE
Add CONTRIBUTING.md file

### ISSUE THREAD

**Issue by vj-codes** [dogecoin/dogecoin.com#196]
### Feature Description
https://github.com/dogecoin/dogecoin has  a contributing.md with all the guidelines
It will better and useful for contributors if this repo also has one

**Comment by vj-codes** [comment]
I would like to work on this if approved:)

**Comment by langerhans** [comment]
Do you think the guides under https://github.com/dogecoin/dogecoin.com/tree/gh-pages/getting-started cover this already?

**Comment by vj-codes** [comment]
@langerhans those are telling about setting up this repo what I meant was a guide for the repo workflow, sqaushing commits, info about pull requests etc 
You can read what I'm speaking about [here ](https://github.com/dogecoin/dogecoin/blob/4272f9bbb18f58915d33b36df2900241f0f5ce50/CONTRIBUTING.md)

**Comment by langerhans** [comment]
I'd be okay with adding something like that. It could also link to these guides as well :)

**Comment by jeremiahisaacson** [comment]
Agreed. It'd be nice to have those guidelines in this repo or a link (less maintenance longer term) under each supported OS as part of the onboarding.

**Comment by NeilShah2026** [comment]
@vj-codes if you need help, let me know, I would love to help

**Comment by vj-codes** [comment]
@NeilShah2026 thank you , I'm almost done will create a PR soon , and you can help by reviewing it :)

**Comment by NeilShah2026** [comment]
👍

**Comment by vj-codes** [comment]
@langerhans I saw the thread on adding license issue and can you please confirm if I can proceed to create a PR for this one

**Comment by vj-codes** [comment]
@langerhans ?^

**Comment by langerhans** [comment]
Sorry been a few very busy days. Just going through the issues and PRs here. Feel free to post a draft of the contribution guide :) The license issue is separate and needs more thought.
